src/client/main.cpp

```diff
@@ -10,24 +10,38 @@
 #include <string>
 #include <unistd.h>
 
+#include <QByteArray>
+
 #include "../shared/ipc.cpp"
+#include "../shared/signature.cpp"
+
+#include "frida/frida-gum.h"
 
 #define FB_ID "mxcfb"
 
 #ifndef _GNU_SOURCE
 #define _GNU_SOURCE
 #endif
 
-using namespace std;
 int msg_q_id = 0x2257c;
 swtfb::ipc::Queue MSGQ(msg_q_id);
 
 uint16_t *SHARED_BUF;
 
+const int SIZE = 2;
```

Comment:
SIZE? seems like it should have better name indicating its bytes per pixel

Reply:
I just copied this, but I'll change it 👍